cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/CaseManager.cs

```diff
@@ -4283,17 +4288,28 @@ public async Task UpdateNonComplyDocuments()
             var dpsUploadedDate = GetDpsUploadedDate();
 
             // This Current Date is applicable to only  ERA results and Road Test - ICBC Re-exam
-            var dpsUploadDateEraResults = GetDpsUplaodDateForERAResults();
+            var dpsUploadDateEraResults = GetDpsUploadDateForERAResults();
+
+            //Path 1 : This is non comply for all the documents except ERA results and Road Test
+
+            UpdateNonComplyDocumentsQuery(dpsUploadedDate);
+
+            // Path 2 : This is to set non comply on  ERA results and Road Test
 
 
+            UpdateNonComplyDocumentsQuery(dpsUploadDateEraResults);
```

Comment:
if I understand this correctly, this method changes the submittal status from all documents open required to 100000005 below the parameter date. If that is true, you could do the same thing by combining to one method call:
```
var maxDate = DateTime.Max(dpsUploadedDate, dpsUploadDateEraResults);
UpdateNonComplyDocumentsQuery(maxDate);
```